avaframe/com1DFA/com1DFA.py

```diff
@@ -1789,12 +1774,27 @@ def DFAIterate(cfg, particles, fields, dem, inputSimLines, simHash=""):
     tCPU["nSave"] = nSave
     nIter = 1
     nIter0 = 1
+    countParticleCsv = 0
     particles["iterate"] = True
     t = particles["t"]
     log.debug("Saving results for time step t = %f s", t)
-    fieldsList, particlesList = appendFieldsParticles(
-        fieldsList, particlesList, particles, fields, resTypesLast
-    )
+
+    # export initial time step
+    if cfg["EXPORTS"].getboolean("exportData"):
+        exportFields1(cfg, t, fields, dem, outDir, cuSimName, TSave="intermediate")
+    # export particles properties for visulation
+    if cfg["VISUALISATION"].getboolean("writePartToCSV"):
+        particleTools.savePartToCsv(
+            cfg["VISUALISATION"]["visuParticleProperties"], [particles], outDir, countParticleCsv=countParticleCsv
+        )
+        countParticleCsv = countParticleCsv + 1
```

Comment:
What is the countParticleCsv needed for? And does this work as expected? Since it is always set to 0 in line 1777

Reply:
in order to be loaded as timeseries in paraview - ascending numbers are required - hence it is just a counter for the files - it is set to zero outside the loop in line 1777 and from thereon one is added for each file saved initial in line 1787 and then inside the time iteration loop